### PR TITLE
HPCC-17157 Document Embed Python3

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-17157 Document Embed Python3
Document embedded Python3 code in HPCCSystems and ECL.
Also addressing HPCC-17223

@JamesDeFabia please review
@Michael-Gardner please review
@richardkchapman please review  and note the link referenced in line 3218, if this is a good link to use. 

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:

Successfully tested building locally.

### CODE REVIEW

File: docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 1]
```diff
@@ -3153,6 +3153,17 @@ add1(10);
       <sect2 id="Add_Python_support" role="brk">
         <title>Python</title>
 
+        <para>HPCC supports Python by default. The HPCC platform includes
```
Comment (Michael-Gardner) [inline docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: HPCC supports Python2 by default.

File: docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 2]
```diff
@@ -3153,6 +3153,17 @@ add1(10);
...
+        plugins for both Python2 and Python3, only one may be safely enabled
```
Comment (Michael-Gardner) [inline docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: ... and Python3, but only one ...

**Review by Michael-Gardner — COMMENTED** [review]
Not sure if these changes would be necessary.  Looks good though.

**Comment by richardkchapman** [comment]
@g-pan Back to you for Michael's comments. Looks good other than that

**Comment by g-pan** [comment]
Added changes, squashed, and rebased.

**Review by JamesDeFabia — COMMENTED** [review]
Looks good from my POV

**Pull request merged**